src/tools/rpc.rs

```diff
@@ -0,0 +1,224 @@
+use std::{
```

Comment:
`rpc.rs` because the go-algorand codebase also calls this API RPC, although this is a REST API 🤷‍♂️
We could rename it to `rest.rs` to reflect the true state

Reply:
RPC is a common name for these types of servers, perhaps maintaining parity with algod is desirable so that it's immediately clear what it corresponds to in the node?